src/krate.rs

```diff
@@ -564,6 +564,10 @@ pub fn index(req: &mut Request) -> CargoResult<Response> {
 
     let recent_downloads = sql::<Nullable<BigInt>>("SUM(crate_downloads.downloads)");
 
+    let not_yanked_versions = sql::<Bool>(
+        "crates.id = ANY (SELECT vs.crate_id FROM versions vs WHERE vs.crate_id = crates.id AND vs.yanked IS FALSE)",
```

Comment:
This could also be `left_join(crates_versions.on(crates::id.eq(crates_versions::crate_id).and(crates_versions::yanked.eq(false)))).filter(crates_versions::id.is_null())`